packages/react/.storybook/manager-head.html

```diff
@@ -1,20 +1,29 @@
-<meta name="keywords" content="IBM, design, system, Carbon, design system, Bluemix, styleguide, style, guide, components, library, pattern, kit, component, cloud, React, React.js">
-<meta name="description" content="A collection of IBM.com components implemented using React and Carbon Design System.">
+<meta
+  name="keywords"
+  content="IBM, design, system, Carbon, design system, Bluemix, styleguide, style, guide, components, library, pattern, kit, component, cloud, React, React.js"
+/>
+<meta
+  name="description"
+  content="A collection of IBM.com components implemented using React and Carbon Design System."
```

Comment:
Is there anything we need to look at in here, or is this just formatting mostly?

Reply:
yep it was just my prettier that reformatted the file after I removed the analytics script from here.